plugins/inara.py

```diff
@@ -96,12 +104,14 @@ def plugin_app(parent):
 
 def plugin_stop():
     # Send any unsent events
-    call()
+    call(force=True)
```

Comment:
I thought we agreed that forcing on close might be bad, as the user *could* conceivably restart EDMC fast enough to then send something else inside the flood limit.

Better to delay/not send the events on close (they can log back in if needs be) than hit the 1-hour timeout because of this.